### ISSUE TITLE
Consider moving the <header> element back inside the header component

### ISSUE THREAD

**Issue by joshuawilson** [openshiftio/openshift.io#937]
_From @pmuir on January 18, 2017 20:58_

https://github.com/fabric8io/fabric8-ui/pull/60 moved it out, but we should consider moving it back in.

_Copied from original issue: fabric8-ui/fabric8-ui#72_

**Comment by joshuawilson** [comment]
I agree, that the header, main, and footer elements belong in the components that generate them. 

@mitchharpur what is the reason to have them outside of their controlling components?

**Comment by joshuawilson** [comment]
@mindreeper2420 @dgutride @dlabrecq what do you guys think about this. Please reference the code and discussion in the PR #60

**Comment by mitchharpur** [comment]
A best practice is to separate site layout (where things go) from components (what things do). The header is part of 'layout', which controls where things go or are placed, while the component is focused on what is being done i.e function is navigation ... two different things and in my opinion do not and should not go together.
